packages/components/nodes/agents/OpenAIAssistant/OpenAIAssistant.ts

```diff
@@ -27,7 +27,7 @@ class OpenAIAssistant_Agents implements INode {
         this.version = 2.0
         this.type = 'OpenAIAssistant'
         this.category = 'Agents'
-        this.icon = 'openai.png'
+        this.icon = 'assistant.svg'
```

Comment:
For this icon, is it possible to add OpenAI logo that cuts into the bot icon? Reason being this is specific to OpenAI only

Reply:
Updated the icon with OpenAI logo